backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/ReduceImageCommand.java

```diff
@@ -276,4 +276,13 @@ public AuditLogType getAuditLogTypeValue() {
     private void addAuditLogCustomValues() {
         this.addCustomValue("DiskAlias", getDiskImage().getDiskAlias());
     }
+
+    @Override
+    public Map<String, String> getJobMessageProperties() {
+        if (jobProperties == null) {
+            jobProperties = super.getJobMessageProperties();
+            jobProperties.put("diskalias", getDiskImage() != null ? getDiskImage().getDiskAlias() : "");
```

Comment:
Can `getDiskImage().getDiskAlias()` return `null`?
What about `N/A` instead of an empty sting? Not sure myself what's better....

Reply:
If I remember correctly, disk should always have an alias.

Reply:
That's why I'm asking :)
https://github.com/oVirt/ovirt-engine/commit/997916583d9c7a2e7bdb7b51c6ae514426752c47

Reply:
> Though maybe the original code should have been updated if getDiskImage().getDiskAlias() can never be null

right, the alias is initialized to an empty string and the validations should ensure it's not null

Reply:
I guess `addAuditLogCustomValues()` just a few lines above is also missing `getDiskImage() != null` check :)